req.py

```diff
@@ -176,159 +213,215 @@ def random_email(name=None):
+                    info = os.getcwd()+"/src/resume.png"
+                case 'addy':
+                    info = fake.street_address()
+                case 'city':
+                    info = city
+                case 'zip':
+                    zipp = zip_codes[city]
+                    info = zipp[zip_num]
+                case 'job':
+                    info = fake.job()
+                case 'salary':
+                    info = random.randint(15, 35)
+
+            driver.find_element(By.XPATH,  data2.get(key)).send_keys(info)
+
+        print(f"Successfully filled out app forms for {city}")
+
+        # fill out dropdowns
+        select = Select(driver.find_element(By.ID,  '154:_select'))
+        select.select_by_visible_text('Yes')
+        select = Select(driver.find_element(By.ID,  '195:_select'))
+        select.select_by_visible_text('United States')
+
+        select = Select(driver.find_element(By.ID,  '211:_select'))
+        select.select_by_visible_text('Yes')
+        select = Select(driver.find_element(By.ID,  '215:_select'))
+        select.select_by_visible_text('No')
+        select = Select(driver.find_element(By.ID,  '219:_select'))
+        select.select_by_visible_text('No')
+        select = Select(driver.find_element(By.ID,  '223:_select'))
+        select.select_by_visible_text('No')
+        select = Select(driver.find_element(By.ID,  '227:_select'))
+        select.select_by_visible_text('No')
+        select = Select(driver.find_element(By.ID,  '231:_select'))
+        select.select_by_visible_text('Yes')
+        select = Select(driver.find_element(By.ID,  '223:_select'))
+        select.select_by_visible_text('No')
+
+        time.sleep(1)
+
+        select = Select(driver.find_element(By.ID,  '235:_select'))
+        gender = random.choice(['Male', 'Female', 'Other'])
+        select.select_by_visible_text(gender)
+
+        driver.find_element(By.XPATH,  '//label[text()="350 LBS"]').click()
+        driver.find_element(By.XPATH,  '//label[text()="800 LBS"]').click()
+        els = driver.find_elements_by_xpath('//label[text()="Yes"]')
+        for el in els:
+            el.click()
 
-        try:
-            generate_account(driver, rand_num)
-        except Exception as e:
-            print(f"failed to create account: {str(e)}")
-            driver.close()
-            continue
+        time.sleep(5)
+        driver.find_element(By.XPATH,  '//*[@id="261:_submitBtn"]').click()
+        print(f"Successfully submitted application")
+        count += 1
+        print(f"Jobs requested: {str(count)}")
+        driver.close()
+    except Exception as e:
+        print(f"Failed to fill out app and submit: {str(e)}")
+        driver.close()
 
-        try:
-            fill_out_application_and_submit(driver, rand_num)
-        except Exception as e:
-            print(f"failed to fill out app and submit: {str(e)}")
-            driver.close()
-            continue
+def main():
+    global processes

```

Comment:
This is the proper way to run multiple things simultaneously. This functionality mirrors what I believe is the intent of https://github.com/SeanDaBlack/KelloggBot/pull/16 , but uses the proper library.